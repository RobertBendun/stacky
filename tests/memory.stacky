# dot compare
1 b1 []byte
b1 511 write8 b1 read8 .

2 b2 []byte
b2 dup 131071 write16 read16 .

4 b3 []byte
b3 dup 8589934591 write32 read32 .

8 b4 []byte
b4 dup 18446744073709551615 write64 read64 .
