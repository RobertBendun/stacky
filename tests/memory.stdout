255
65535
4294967295
18446744073709551615
